<?xml version="1.0" encoding="utf-8" ?>

<launch>
    <arg name="mode" default="$(env MOVEL_MODE)" doc="mode: [deployment, development]"/>

    <group if="$(eval mode == 'deployment')">
        <node pkg="movel_fms_utils" type="plan_decimator_node" name="plan_decimator_node" output="screen">
            <rosparam file="$(find movel_fms_utils)/config/plan_decimator.yaml" command="load" />
            <rosparam file="$(find movel)/config/move_base_params.yaml" ns="move_base_params" />
            <rosparam file="$(env HOME)/.config/movel/config/movel_fms_utils/config/plan_decimator.yaml" command="load"/>
            <rosparam file="$(env HOME)/.config/movel/config/movel/config/move_base_params.yaml" ns="move_base_params"/>
        </node>
    </group>

    <group unless="$(eval mode == 'deployment')">
        <node pkg="movel_fms_utils" type="plan_decimator_node" name="plan_decimator_node" output="screen">
            <rosparam file="$(find movel_fms_utils)/config/plan_decimator.yaml" command="load" />
            <rosparam file="$(find movel)/config/move_base_params.yaml" ns="move_base_params" />
        </node>
    </group>
</launch>
